### PR TITLE
Add New York, USA provider

### PR DESCRIPTION
**Pull request by chimo**
Ref. #84, grote/Transportr#180

### CODE REVIEW

File: enabler/src/de/schildbach/pte/NewyorkProvider.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * @author Stephane Berube
+ */
+public class NewyorkProvider extends AbstractNavitiaProvider
+{
+	private static String API_REGION = "us-ny";
+
+	public NewyorkProvider(final String apiBase, final String authorization)
+	{
+		super(NetworkId.QUEBEC, apiBase, authorization);
```
Comment (grote) [inline enabler/src/de/schildbach/pte/NewyorkProvider.java]: It probably should not use `QUEBEC` here.

File: enabler/src/de/schildbach/pte/NewyorkProvider.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+		setTimeZone("America/Montreal");
```
Comment (grote) [inline enabler/src/de/schildbach/pte/NewyorkProvider.java]: This should probably say `New_York` as well.

**Comment by chimo** [comment]
@grote thanks for the comments. Guess I was tired when I did this :|

**Comment by leandroferreira** [comment]
@grote @chimo Is there any additional work required on this PR before it can be merged?

**Comment by grote** [comment]
Not that I know of. You can practically test this in Transportr already and let us know how well it works. @schildbach is the only one that can merge PRs here.

**Pull request closed without merging**